### DISCUSSION TITLE
Welcome to ngx-matomo Discussions!

### DISCUSSION THREAD

**Discussion by Arnaud73** [Arnaud73/ngx-matomo#51] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mochinikunj** [comment]
how to track multiple routes in angular? and How can I download v1.X.X ? 

Using "npm i ngx-matomo" command I'm able to download v0.1.4 and when I downloaded using "npm i ngx-matomo@1.0.0-rc.2", I was able to download pre-release, but then it failed to compile and run my application.

I'm using angular v11 for my project.

**Comment by ROKKAM-SUSMITHA** [comment]
I am using angular 7 and i have installed ngx-matomo version 0.0.3 but while compiling i am getting error like : Property 'forRoot' does not exist on type 'typeof MatomoModule'.ts(2339)
